kiwi_post_run

```diff
@@ -45,6 +42,9 @@ CONTAINER_TAG=$( xmllint --xpath "string(//image/preferences/type/containerconfi
                     $TOPDIR/KIWIROOT-docker/image/config.xml )
```

Comment:
I assume that the xmllint command is the one failing and causing the script to exit with a non-zero code, right?

If so I am wondering how is it exiting with the change you suggest. I guess it exits because some of the later checks are not passing right? ($IMAGE or $CONTAINER_TAG). If so I am wondering why aren't we introducing a new test instead of ignoring some errors and expect the script to detect there is some wrong at later stages.

We could include something like

```bash
if [ ! -f $TOPDIR/KIWIROOT-docker/image/config.xml ]; then
    echo "Could not find config.xml file!"
    echo "Will skip containment rpm wrapping steps."
    exit 0
fi
```

before running xmllint and keep `set -e` and `set -u` where they are.